tests/Couchbase.EntityFrameworkCore.FunctionalTests/ContosoUniversityTests/StudentTests.cs

```diff
@@ -61,94 +62,94 @@ public async Task Test_Get(string sortOrder,
 
         int pageSize = 3;
         var paginatedList =
-            await PaginatedList<Student>.CreateAsync(students.AsNoTracking(), pageNumber ?? 1, pageSize);
-        
+            await PaginatedList<Student>.CreateAsync(students.AsNoTracking(), pageNumber ?? 1,
+                pageSize);
+
         Assert.Equal(8, paginatedList.Count);
     }
 
     [Fact]
     public async Task Test_PersonStudent_AddRange()
     {
-        var context = _contosoFixture.DbContext;
-        context.Students.AddRange(_students);
+        var context = _contosoFixture.DbContext();
```

Comment:
SInce the DbContext has a Dispose method, you should wrap this in a using statement or explicitly call Dispose in the finally block of a trycatch. There are a few other places this needs to be done as well.

Reply:
Good catch.